### PR TITLE
Update the generic issue template

### PR DESCRIPTION
**Pull request by iherman**
Let us see if this works...

Using this template it may be possible to add a separate template for [BUG] or [FEATURE REQUEST]; with the volume of issues I am not sure if it is worth doing...

(Wendy, Shinya: we regularly get issues that are irrelevant for this repo, and we have to close. Matt and I wondered if it helps to set up an issue template to deter people from doing so; this is this PR. Note: I have never created issue templates, I hope it will work...)

### CODE REVIEW

**Comment by mattgarrish** [comment]
I wonder if listing all the documents might be overload in an issue. Maybe we could shorten it to something like: "This repository is for reporting issues in the standards and notes produced by the Publishing Maintenance Working Group."

Should we provide any questions to guide users after the text? I don't think it has to be a lot, but maybe:

- Which standard or note is this issue related to? (For bug reports, include the relevant section.)
- Describe the problem? (For bug reports, cite the problematic text.)
- Describe the fix or new feature you propose?

It might help reinforce that we don't want how-to questions here.

**Comment by iherman** [comment]
> I wonder if listing all the documents might be overload in an issue. Maybe we could shorten it to something like: "This repository is for reporting issues in the standards and notes produced by the Publishing Maintenance Working Group."

... in which case people may (a) try to find the references to the documents in the PM WG and (b) submit comments on audiobooks or the publishing manifest...

I think it is better to keep the list. It saves the reader to go to step (a) and make the mistake of (b).

> 
> Should we provide any questions to guide users after the text? I don't think it has to be a lot, but maybe:
> 
> * Which standard or note is this issue related to? (For bug reports, include the relevant section.)
> * Describe the problem? (For bug reports, cite the problematic text.)
> * Describe the fix or new feature you propose?
> 
> It might help reinforce that we don't want how-to questions here.

It might be a simple version of the question I had at the beginning (having separate issue templates for bugs/features).

Do you think you could make an inline comment with the question? It might make it simpler to process...

**Comment by iherman** [comment]
B.t.w., the magic of setting up issue templates seems to be, simply, to have markdown files in the `.github/ISSUE_TEMPLATE/` directory, with some general metadata at the top. I presume anyone with the right to merge/commit can do that. I have learned something, again, about github :-)

**Comment by mattgarrish** [comment]
> I think it is better to keep the list.

I'm just worried that it will push the important part we want people to read off screen. The input box isn't all that big, so someone might just read that it's for issues with the following specs and still presume that means any how-to question.

Could we can merge the paragraph after the list into the top one? So maybe have:

> This repository is only for issues with the following W3C documents. Questions or issues with specific reading systems, achieving specific styling, understanding validation errors, and similar will be considered as out of scope and will be closed without further action.

**Comment by iherman** [comment]
> > I think it is better to keep the list.
> 
> I'm just worried that it will push the important part we want people to read off screen. The input box isn't all that big, so someone might just read that it's for issues with the following specs and still presume that means any how-to question.
> 
> Could we can merge the paragraph after the list into the top one? So maybe have:
> 
> > This repository is only for issues with the following W3C documents. Questions or issues with specific reading systems, achieving specific styling, understanding validation errors, and similar will be considered as out of scope and will be closed without further action.

Yep, I like that; I've made the changes.

File: .github/ISSUE_TEMPLATE/general-issue-template.md
Comment (mattgarrish) [inline .github/ISSUE_TEMPLATE/general-issue-template.md:10]: If we're going to add questions, which I'll pose in another comment, maybe we should put a heading here.
```suggestion
## Before opening an issue
This repository is only for issues with the following W3C documents. Questions or issues with specific reading systems, achieving specific styling, understanding validation errors, and similar will be considered as out of scope and will be closed without further action.
```

File: .github/ISSUE_TEMPLATE/general-issue-template.md
Comment (mattgarrish) [inline .github/ISSUE_TEMPLATE/general-issue-template.md:40]: It might be useful to add starter questions.
```suggestion
## Which standard or note is this issue related to?

Please include the title and version number. For bug reports, include the relevant section.

## Describe the problem? 

What is not working? What would you like to see improved? For bug reports, cite the relevant text from the specification.

## Describe the fix or new feature you propose?

```

**Comment by iherman** [comment]
@wareid, @shiestyle, could you give your blessing (or opposition to) this? Just to have at least another pair of eyes...

**Review by shiestyle — APPROVED** [review]
I agree with this update, let's try.

**Comment by iherman** [comment]
I will go ahead and merge this (and see if it works). We can always change it later, the real question is whether it works as intended...

**Comment by mattgarrish** [comment]
Looks good, although seeing it in action I now wonder if we should have done separate feature and bug templates. The generic template doesn't load by default, so you still have to pick it to begin.

**Pull request merged**
